lib/vsc/install/shared_setup.py

```diff
@@ -1316,6 +1325,23 @@ def get_license(self, license_name=None):
         log.info("Found license name %s and classifier %s", lic_short, data[1])
         return lic_short, data[1]
 
+    def locate_readme(self):
+        """
+        Returns path to README file in the package from allowed options in README global
+        The highest option in README global is preferred
```

Comment:
fyi, the order is README random. but not sure it matters much

Reply:
you are right, I was not thinking about Python < 3.6. I moved it to a tuple so that the order of preference is always predictable.